### PR TITLE
Change math fidelity from hifi2 to lofi in case we use bfp8_b, WIP

### PR DESCRIPTION
**Pull request by vcanicTT**

### CODE REVIEW

File: benchmark/models/flant5/flant5_past_cache_enc_dec.py [hunk 1]
```diff
@@ -83,6 +83,11 @@ def flant5_past_cache_enc_dec(training: bool, task: str, config: str, microbatch
         compiler_cfg.compile_subgraphs = True
         compiler_cfg.enable_link_past_cache_ios = True
 
+        # if data_type == "Bfp8_b":
```
Comment (milank94) [inline benchmark/models/flant5/flant5_past_cache_enc_dec.py:86]: Remove any commented out code.

File: benchmark/models/mobilenet_v1/mobilenet_v1.py [hunk 2]
```diff
@@ -108,6 +116,23 @@ def eval_fn(**kwargs):
 
         # Define evaluation function
         def eval_fn(outputs, labels):
+
+            print("\n")
```
Comment (milank94) [inline benchmark/models/mobilenet_v1/mobilenet_v1.py:120]: I think this all needs to be removed. Looks like debug code.

File: benchmark/models/mobilenet_v2/mobilenet_v2.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 from transformers import AutoImageProcessor, AutoModelForImageClassification
 
 from ...common import DummyCVDataset, ImageNetDataset, benchmark_model, torch_df_from_str
+from pybuda.config import _get_global_compiler_config
```
Comment (milank94) [inline benchmark/models/mobilenet_v2/mobilenet_v2.py:12]: Similar comment, please keep this under `if device == "tt":`

File: benchmark/models/unet/unet.py [hunk 4]
```diff
@@ -9,34 +9,35 @@
 from tqdm import tqdm
 
 from ...common import BrainSegmentationDataset, DummyCVDataset, benchmark_model, torch_df_from_str
+from pybuda.config import _get_global_compiler_config
```
Comment (milank94) [inline benchmark/models/unet/unet.py:12]: Similar comment, please keep this under `if device == "tt":`

File: benchmark/models/unet/unet.py [hunk 5]
```diff
@@ -9,34 +9,35 @@
...
+        # These are about to be enabled by default.
+        #
+        os.environ["PYBUDA_TEMP_ENABLE_NEW_FUSED_ESTIMATES"] = "1"
+        if data_type == "Fp16_b":
+            os.environ["PYBUDA_TEMP_ENABLE_NEW_SPARSE_ESTIMATES"] = "1"
+
         if data_type == "Bfp8_b":
-            pybuda.config.configure_mixed_precision(op_type="matmul", output_df=pybuda.DataFormat.Float16_b)
-            pybuda.config.configure_mixed_precision(op_type="add", output_df=pybuda.DataFormat.Float16_b)
+            # pybuda.config.configure_mixed_precision(op_type="matmul", output_df=pybuda.DataFormat.Float16_b)
```
Comment (milank94) [inline benchmark/models/unet/unet.py:38]: Remove any commented out code.

**Comment by vcanicTT** [comment]
This PR is not relevant anymore. I'm closing it!

**Pull request closed without merging**